Add changelog (#109)
remarkjs/vscode-remark: changelog.md

# Changelog

> Please see our [GitHub releases](https://github.com/remarkjs/vscode-remark/releases) to find out what’s new.

Reviewer: ```suggestion
See [GitHub Releases][releases] for the changelog.

[releases]: https://github.com/remarkjs/vscode-remark/releases
```

This matches remark and similar
